src/Components/WebAssembly/Build/src/targets/StaticWebAssets.targets

```diff
@@ -1,37 +1,161 @@
 <Project>
 
-  <PropertyGroup>
-    <ResolveStaticWebAssetsInputsDependsOn>
-      $(ResolveStaticWebAssetsInputsDependsOn);
-      _RemoveBlazorCurrentProjectAssetsFromStaticWebAssets;
-    </ResolveStaticWebAssetsInputsDependsOn>
+  <Target Name="_ResolveBlazorGeneratedAssets" DependsOnTargets="PrepareBlazorOutputs">
+    <ItemGroup>
+      <StaticWebAsset Include="@(BlazorOutputWithTargetPath->'%(FullPath)')" RemoveMetadata="TargetOutputPath">
+        <SourceType></SourceType>
+        <SourceId>$(PackageId)</SourceId>
+        <ContentRoot>$([MSBuild]::NormalizeDirectory('$(TargetDir)wwwroot\'))</ContentRoot>
+        <BasePath>$(StaticWebAssetBasePath)</BasePath>
+        <RelativePath>$([System.String]::Copy('%(BlazorOutputWithTargetPath.TargetOutputPath)').Replace('dist\','').Replace('\','/'))</RelativePath>
```

Comment:
Just want to check, is there any chance the ordering of `Replace('dist\','')` and `Replace('\','/')` should be reversed?

As it is, we would remove `dist\` but *not* `dist/`. Is that intentional?

Reply:
Good catch, we do this in a few places, I'll change it.

Reply:
Thanks. Do you happen to know what test coverage we're missing such that this didn't fail any tests before?

Reply:
It actually means it doesn't matter. It is likely MSBuild normalized the path here to '/dist' in non windows platforms.